### ISSUE TITLE
Prevent line break before ; when exceeding line length by one character

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#26]
If a line has 101 characters with a max line length setting of 100 the following will happen:

```
// in
fSomeVariableName1 := LIMIT(fTheLowerLimitVariable, fSomeOtherVariableName, fTheUpperLimitVariable);

// out
fSomeVariableName1 := LIMIT(fTheLowerLimitVariable, fSomeOtherVariableName, fTheUpperLimitVariable)
	;
```

Better alternatives would be to either not strictly adhere to the max line length

```
// in
fSomeVariableName1 := LIMIT(fTheLowerLimitVariable, fSomeOtherVariableName, fTheUpperLimitVariable);

// out
fSomeVariableName1 := LIMIT(fTheLowerLimitVariable, fSomeOtherVariableName, fTheUpperLimitVariable);
```    
Or break up the fb call as set in the settings when it exceeds 101 characters.

```
// in
fSomeVariableName1 := LIMIT(fTheLowerLimitVariable, fSomeOtherVariableName, fTheUpperLimitVariable);

// out
fSomeVariableName1 := 
    LIMIT(fTheLowerLimitVariable, fSomeOtherVariableName, fTheUpperLimitVariable);
```

**Comment by Barteling** [comment]
I agree that that line breaking after a semicolon does not make much sense, 
That would be a minor change for me to implement, but I have to investigate if this true for all situations.

I don't think working with a more flexible line limit is a good solution. The limit should be the limit. I believe that keeps it more clear.

**Comment by Barteling** [comment]
Fixed with 3.1.0Beta
